src/app/@ansyn/menu-items/tools/reducers/tools.reducer.ts

```diff
@@ -28,6 +28,7 @@ export interface IToolsState {
 	annotationMode: AnnotationMode;
 	annotationProperties: AnnotationProperties
 	manualImageProcessingParams: ImageManualProcessArgs;
+	imageProcessingHash: {};
```

Comment:
type of `imageProcessingHash` ? 
maybe: `{ [key: string]: ImageManualProcessArgs } `